qiskit_ibm_provider/ibm_provider.py

```diff
@@ -594,6 +595,7 @@ def jobs(
             descending: If ``True``, return the jobs in descending order of the job
                 creation date (i.e. newest first) until the limit is reached.
             instance: The provider in the hub/group/project format.
+            legacy: Retrieve jobs run from the deprecated `qiskit-ibmq-provider`.
```

Comment:
```suggestion
            legacy: If ``True``, only retrieve jobs run from the deprecated ``qiskit-ibmq-provider``. Otherwise retrieve only jobs run from ``qiskit-ibm-runtime``.
```